Empty typescript files lead to tsc assertion

Trying to compile an empty ts file leads to:

```
error: AssertionError: "data" is unexpectedly null for "file:///mnt/f9/Projects/github.com/denoland/deno_lint/foo.ts".
    at assert (deno:cli/tsc/99_main_compiler.js:53:13)
    at Object.getSourceFile (deno:cli/tsc/99_main_compiler.js:377:9)
    at findSourceFile (deno:cli/tsc/00_typescript.js:104355:29)
    at deno:cli/tsc/00_typescript.js:104232:85
    at getSourceFileFromReferenceWorker (deno:cli/tsc/00_typescript.js:104199:34)
    at processSourceFile (deno:cli/tsc/00_typescript.js:104232:13)
    at processRootFile (deno:cli/tsc/00_typescript.js:104055:13)
    at deno:cli/tsc/00_typescript.js:102979:60
    at Object.forEach (deno:cli/tsc/00_typescript.js:382:30)
    at Object.createProgram (deno:cli/tsc/00_typescript.js:102979:16)
```

--- a/cli/tests/empty_typescript.out
+++ b/cli/tests/empty_typescript.out
@@ -1,0 +1,1 @@
+[WILDCARD]

--- a/cli/tests/empty_typescript.out
+++ b/cli/tests/empty_typescript.out
@@ -1,1 +1,0 @@
-[WILDCARD]

--- a/cli/tests/integration_tests.rs
+++ b/cli/tests/integration_tests.rs
@@ -2275,7 +2275,7 @@ itest!(config {
 
 itest!(emtpy_typescript {
   args: "run --reload subdir/empty.ts",
-  output: "empty_typescript.out",
+  output_str: Some("Check file:[WILDCARD]tests/subdir/empty.ts\n"),
 });
 
 itest!(error_001 {

--- a/cli/tests/integration_tests.rs
+++ b/cli/tests/integration_tests.rs
@@ -2273,6 +2273,11 @@ itest!(config {
   output: "config.ts.out",
 });
 
+itest!(emtpy_typescript {
+  args: "run --reload subdir/empty.ts",
+  output: "empty_typescriot.out",
+});
+
 itest!(error_001 {
   args: "run --reload error_001.ts",
   exit_code: 1,

--- a/cli/tests/integration_tests.rs
+++ b/cli/tests/integration_tests.rs
@@ -2275,7 +2275,7 @@ itest!(config {
 
 itest!(emtpy_typescript {
   args: "run --reload subdir/empty.ts",
-  output: "empty_typescriot.out",
+  output: "empty_typescript.out",
 });
 
 itest!(error_001 {

--- a/cli/tsc/99_main_compiler.js
+++ b/cli/tsc/99_main_compiler.js
@@ -374,7 +374,10 @@ delete Object.prototype.__proto__;
           "op_load",
           { specifier },
         );
-        assert(data, `"data" is unexpectedly null for "${specifier}".`);
+        assert(
+          data != null,
+          `"data" is unexpectedly null for "${specifier}".`,
+        );
         sourceFile = ts.createSourceFile(
           specifier,
           data,

